cmd/config.go

```diff
@@ -309,6 +301,85 @@ type CAADistributedResolverConfig struct {
 	Proxies     []string
 }
 
+type TemporalLogDescription struct {
+	URI         string
+	Key         string
+	WindowStart string
```

Comment:
Can you add a comment describing `WindowStart`/`WindowEnd` as being in RFC3339 format?

Reply:
Also, let's just make this be of type `time.Time`, then `json.Unmarshal` can [parse it automatically](https://godoc.org/time#Time.MarshalJSON), so we don't have a string representation and then a separate time representation.

Reply:
TIL